src/index.html

```diff
@@ -1,14 +1,24 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+
+  <body>
+    <div class="clock">
+      <div class="clock__top"></div>
+      <div class="clock__right"></div>
+      <div class="clock__bottom"></div>
+      <div class="clock__left"></div>
+      <div class="clock__center"></div>
+      <div class="clock__shadow"></div>
+      <div class="clock__minute"></div>
+      <div class="clock__second"></div>
+    </div>
```

Comment:
Checklist p.1 - Don't forget about semantics, div development is not a better option
This is the only problem I found there)

Reply:
done